### PR TITLE
Add a section on guardians

### LINKED ISSUE #157: Add a section on the rights of children and how to balance them with the rights of their parents
There is a wide variety of surveillance technology today intended for use on children, by parents or other authorities. The vast majority of public-facing material which addresses the ethics around this technology does so purely with regard to the rights of *parents* to be notified, to consent, etc. Unfortunately, that's pretty much what the law says in most jurisdictions: Most legal rights pertaining to children are actually rights that their parents have.

As a queer person, I do not know that I would have survived to adulthood if the surveillance technology that exists today had been around when I was young. I think it would be amazing for this spec to have a section describing the concerns that can arise where rights of parents come into conflict with legal or moral rights of children, and to present a vocabulary that can be used when balancing those concerns.

This idea came up during a Twitter conversation, and I'm writing this issue in an attempt to capture the thought. Please feel free to ask clarifying questions and I'll try to elaborate. I also take no strong position on exactly what form the proposed write-up should take; I'm just trying to flag a gap that I see in what's there now.

### PR DESCRIPTION
**Pull request by jyasskin**
Intended to fix #157.

@IreneKnapp, how does this look?

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/203.html#guardians" title="Last updated on Dec 14, 2022, 5:08 PM UTC (e77bd36)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/203/94428dc...jyasskin:e77bd36.html" title="Last updated on Dec 14, 2022, 5:08 PM UTC (e77bd36)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1545,6 +1545,46 @@
...
+misbehaving guardian.
+
+Historically, the Web has provided exactly this escape route, and [=user agents=] should preserve
+that feature by correctly balancing a benevolent [=guardian=]'s need to protect their [=ward=] from
+dangers against other [=wards=]' need to protect themselves from their dangerous [=guardians=].
+
+<aside class="example" id="example-protective-parent" title="Protective parents">
+
+A parent might configure a small child's [=user agent=] to block access to violent content until the
+child is old enough to make their own decisions about it.
```
Comment (jyasskin) [inline index.html:1571]: @torgo, I feel like you had a good example of this sort of protection. What was it?

**Comment by IreneKnapp** [comment]
It looks like a good high-level principle, thank you very much!

File: index.html [hunk 2]
```diff
@@ -1602,6 +1602,57 @@
...
+
+</aside>
+
+<aside class="example" id="example-lgbt-kid" title="An LGBT child">
+
+A child may discover that they're LGBT and need to find supportive resources online. If they have a
+homophobic or transphobic parent, that parent might have configured their [=user agent=] to either
+block or inform the parent when the child visits pro-LGBT websites. The [=user agent=] needs to warn
```
Comment (dmarti) [inline index.html:1644]: minor edit -- "web pages about LGBT-related subjects" instead of just pro-LGBT? (filters could also apply to NPOV sites)

Comment (jyasskin) [reply index.html]: Done.

**Review by torgo — APPROVED** [review]
lgtm

**Pull request merged**